frontend/src/components/common/form/StepForm/StepperField.tsx

```diff
@@ -0,0 +1,20 @@
+import Stepper from 'components/common/Stepper';
+import { useFormikContext, getIn } from 'formik';
+import React from 'react';
+import { IStepperFieldProps } from '.';
+
+/**
+ * Using a stepper as a formik field
+ * @component
+ * @example ./StepperField.md
+ */
+export const StepperField: React.FC<IStepperFieldProps> = ({ name, steps }) => {
```

Comment:
I don't understand what a StepperField is?

Reply:
This is a stepper component wrapped with a formik field, to enable the stepper to controlled using formik actions